x-pack/package.json

```diff
@@ -143,6 +160,7 @@
     "apollo-link-state": "^0.4.1",
     "apollo-server-errors": "^2.0.2",
     "apollo-server-hapi": "^1.3.6",
+    "awesome-typescript-loader": "^5.2.1",
```

Comment:
I looked up this package and looks like it's similar to ts-loader?  Could we get away with only one, or does this come with benefits?  Just thinking in the mindset of trying to minimize our dependencies.

Reply:
Let me test it with just `ts-loader`... I seem to recall documentation having a problem, but I may have resolved that.